helm/distributed-idol/values.yaml

```diff
@@ -79,11 +91,13 @@ content:
     version: "23.4"
```

Comment:
Should update these versions whilst here.

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/80/commits/624d2207ef4d5e4bfa38b6ea310b6b990672dc06